proxy/src/http.rs

```diff
@@ -246,12 +247,23 @@ where
         .boxed()
 }
 
+/// Similar to [`warp::filters::query::query`] but provides the default value if
+/// the query string cannot be parsed.
+#[must_use]
+pub fn query_default<T>() -> BoxedFilter<(T,)>
+where
+    for<'de> T: Deserialize<'de> + Default + Send + Sync + 'static,
```

Comment:
Instead of using `Default` can we pass in the default as a parameter? I find that more useful than (sometimes) arbitrary default impls.

Reply:
Fixed in #791

Reply:
Should this not be removed because of #791?